cmd/zetae2e/local/local.go

```diff
@@ -208,6 +208,8 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 	if !skipSetup {
 		noError(deployerRunner.FundEmissionsPool())
 	}
+	// Run the proposals under the start sequence(proposals_e2e_start folder)
+	noError(deployerRunner.CreateGovProposals(runner.StartOfE2E))
```

Comment:
I think I would mildly prefer if this was restricted to the admin tests

Reply:
why admin tests ? it seems unrelated

Reply:
I would prefer if we didn't spend time doing this during normal test execution.

If you want to run this during the normal tests, then please do not run them during all the other targets (performance, upgrade, ton, legacy, admin, etc)

Reply:
okay added skipregular check